adapters/oidc/installed/src/main/java/org/keycloak/adapters/installed/KeycloakInstalled.java

```diff
@@ -77,6 +69,7 @@
  */
 public class KeycloakInstalled {
     private static final String KEYCLOAK_JSON = "META-INF/keycloak.json";
+    private static final String LOGGED_OUT = "loggedout";
```

Comment:
```suggestion
    private static final String LOGGED_OUT = "logout";
```

Nit pick, but `loggedout` just doesn't feel right ;)